lld/ELF/SyntheticSections.h

```diff
@@ -796,6 +796,15 @@ class RelroPaddingSection final : public SyntheticSection {
   void writeTo(uint8_t *buf) override {}
 };
 
+class ShufflePaddingSection final : public SyntheticSection {
```

Comment:
We can always rename the section if it turns out to be needed for another purpose. For now let's name it after its only user so that it's clear what it's used for.